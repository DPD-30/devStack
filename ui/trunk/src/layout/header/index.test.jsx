import React from 'react'
import { renderToString } from 'react-dom/server'


import LayoutHeader from './index.jsx'

jest.unmock('./index.jsx')


test('LayoutHeader matches snapshot', () => {
	const comp = renderToString(<LayoutHeader />)

	expect(comp).toMatchSnapshot()
})
