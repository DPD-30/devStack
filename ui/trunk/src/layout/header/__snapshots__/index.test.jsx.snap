// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutHeader matches snapshot 1`] = `
<div style="height:134px;width:100%;background:url(header.jpg);background-repeat:no-repeat;background-size:cover"
     data-reactroot
>
  <div style="padding:1.3em 1em;font-size:32px;font-weight:bold;color:#19476A">
    Header
  </div>
</div>
`;
